lib/SILGen/SILGenLValue.cpp

```diff
@@ -2663,8 +2663,8 @@ namespace {
     }
 
     void emitUsingAccessor(AccessorKind accessorKind, bool isDirect) {
-      auto accessor =
-        SGF.SGM.getAccessorDeclRef(Storage->getOpaqueAccessor(accessorKind));
```

Comment:
`getOpaqueAccessor` will ignore synthesized declarations, and return nullptr in some cases. `getSynthesizedAccessor` will always return an accessor, even generating one on the fly is necessary. This will pick up the synthesized read accessor, while `getOpaqueAccessor` can ignore it.